Dispatcher/UI/Onboarding/OnboardingScreenViewController.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  OnboardingScreenViewController.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 01/06/2023.
+//
+
+import UIKit
+
+class OnboardingScreenViewController: UIViewController {
+
+    @IBOutlet weak var dispatcherLabel: UILabel!
+    @IBOutlet weak var boardingProgressView: UIProgressView!
+    @IBOutlet weak var introductionLabel: UILabel!
+    @IBOutlet weak var nextButton: UIButton!
+    @IBOutlet weak var skipButton: UIButton!
+    @IBOutlet weak var backgroundImageView: UIImageView!
+    
+    var currentPage = 1
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        introductionLabel.text = TextCostants.firstOnBaordingText
+        boardingProgressView.setProgress(0.333, animated: true)
+        boardingProgressView.layer.cornerRadius = 7
+    }
+    
+
+    @IBAction func skipButtonPressed(_ sender: UIButton) {
```

Comment:
Why does "skip" bring the user to the 3rd onboarding page? Should move him to the auth screen